.circleci/config.yml

```diff
@@ -16,6 +16,42 @@ commands:
             cd ~/amplify-android-home
             yes | sdkmanager --licenses && yes | sdkmanager --update
             ~/gradle-5.6.2/bin/gradle wrapper
+  configure_aws:
+    description: >-
+      install aws cli and configure android aws release profile
+    steps:
+      - run:
+          name: install aws cli
+          command: |
+            sudo pip install awscli
+      - run:
+          name: configure aws profile
+          command: |
+            aws configure --profile amplify_sdk_test set region us-east-1
```

Comment:
Would recommend testing with us-west-1 or us-west-2 as it is closer.

Reply:
Sounds good. Will udpate